test/evergreen.yml

```diff
@@ -2067,6 +2067,35 @@ tasks:
             cp ../../../bench/wtperf/split_heavy.wtperf .
             ./wtperf -O ./split_heavy.wtperf -o verbose=2
 
+  - name: static-wt-build-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-static
```

Comment:
@agorrod could you please confirm if the `--disable-shared --with-builtins=snappy,zlib` configure settings pasted in Jira are essential (or optional) for this test?

Reply:
They are necessary. Without those options the compile will build a shared library, when this test is explicitly about a static library.

Reply:
@lukech & @agorrod : I think "--with-builtins=snappy,zlib" is not necessary here because "wt" utility in this test is not used for any compression, it is just used to print the wiredtiger library version and validates if "wt" utility build is static. 
But I am open for suggestions :)

Reply:
I agree that it's not confirmed to work by the test, but it is valuable to be doing the static build with that option. The reason we have this is to ensure we can compile a `wt` binary that can be sent as a single file and used to investigate issues with MongoDB deployments.

Reply:
Thank you @agorrod , updated to configure with snappy & zlib options.